### ISSUE TITLE
Alternative image text missing

### ISSUE THREAD

**Issue by mnaibei** [kalbek/Portfolio#17]
Your code is well structured and clean. However I have noticed you are missing some [alt tags](https://github.com/kalbek/Portfolio/blob/45b475cd91fdd53fd8515d22d04e85e08972a332/index.html#L49-L57) on your images. Please add them.

**Comment by kalbek** [comment]
I agree the first alt text should be "see live icon " and the second one should also be "see source icon"
